### PR TITLE
bumping async storage

### PR DESCRIPTION
**Pull request by GregoryCollett**
@react-native-community/async-storage is now deprecated.

### CODE REVIEW

**Comment by morrys** [comment]
Hi @GregoryCollett, thanks for your PR.

Since even expo is now compatible with react-native-async-storage/async-storage
https://docs.expo.io/versions/latest/sdk/async-storage/

I would change [createStorage.native.ts](https://github.com/morrys/wora/blob/c9da9c68dd4295051a0dd1558d12e7e2c223e20d/packages/cache-persist/src/createStorage.native.ts) like this:

```ts
import { ICacheStorage } from './CacheTypes';
import AsyncStorage from '@react-native-async-storage/async-storage';

/**
 * I had to restore the abstraction above the AsyncStorage for a
 * typescript problem in the multiGet (readonly parameter)
 *
 */

export function createStorage(_type): ICacheStorage {
    return {
        multiRemove: (keys) => AsyncStorage.multiRemove(keys),
        multiGet: (keys) => AsyncStorage.multiGet(keys),
        getAllKeys: () => AsyncStorage.getAllKeys(),
        multiSet: (items) => AsyncStorage.multiSet(items),
        setItem: (key, value) => AsyncStorage.setItem(key, value),
        removeItem: (key) => AsyncStorage.removeItem(key),
        getItem: (key) => AsyncStorage.getItem(key),
    } as ICacheStorage;
}
```

What do you think about it?

**Comment by GregoryCollett** [comment]
Hi @morrys,

Great shout, I have updated the pr to reflect this change.

File: packages/cache-persist/src/createStorage.native.ts [hunk 1]
```diff
@@ -1,20 +1,5 @@
-/* eslint import/no-extraneous-dependencies:0 */
```
Comment (morrys) [inline packages/cache-persist/src/createStorage.native.ts:1]: I think this is necessary 👍

Comment (GregoryCollett) [reply packages/cache-persist/src/createStorage.native.ts:1]: On that note, is there a reason this is only listed as a development dependency?

File: packages/cache-persist/package.json [hunk 2]
```diff
@@ -33,10 +33,10 @@
     "test": "jest --coverage"
   },
   "dependencies": {
-    "idb": "^4.0.0"
+    "idb": "^4.0.0",
+    "@react-native-async-storage/async-storage": "1.13.3"
```
Comment (morrys) [inline packages/cache-persist/package.json:37]: I prefer it to be handled as devDependencies as the package is web-first 👍

Comment (GregoryCollett) [reply packages/cache-persist/package.json:37]: Makes sense! Will rectify and reinstate the lint comment! Thanks for explaining!

File: yarn.lock [hunk 3]
```diff
@@ -5,15 +5,13 @@
 "@apollo/react-common@^3.1.3":
   version "3.1.3"
   resolved "https://registry.yarnpkg.com/@apollo/react-common/-/react-common-3.1.3.tgz#ddc34f6403f55d47c0da147fd4756dfd7c73dac5"
-  integrity sha512-Q7ZjDOeqjJf/AOGxUMdGxKF+JVClRXrYBGVq+SuVFqANRpd68MxtVV2OjCWavsFAN0eqYnRqRUrl7vtUCiJqeg==
```
Comment (morrys) [inline yarn.lock:8]: this is strange... can i ask which version of yarn are you using?

Comment (GregoryCollett) [reply yarn.lock]: I have rebuilt this with the latest, I had auto env resolving to a much older release (1.9.4).

File: yarn.lock [hunk 4]
```diff
@@ -2134,7 +2136,7 @@
 "@wora/apollo-cache@file:packages/apollo-cache":
   version "2.2.0"
   dependencies:
-    "@wora/cache-persist" "file:packages/cache-persist"
+    "@wora/cache-persist" "file:../../Library/Caches/Yarn/v6/npm-@wora-apollo-cache-2.2.0-1e7edac8-41ff-46b5-a71d-4fe16e9444c1-1611579151904/node_modules/@wora/cache-persist"
```
Comment (GregoryCollett) [inline yarn.lock:2139]: These look incorrect but is what yarn is throwing out at me. 🤔

Comment (morrys) [reply yarn.lock:2139]: That is a known issue: https://github.com/yarnpkg/yarn/issues/4388 😅

**Comment by GregoryCollett** [comment]
Thanks!

**Comment by morrys** [comment]
thank you for your contribution 💯 

I will release the libraries as soon as possible.

Can I ask you which library are you using?

**Pull request merged**